src/switchcraft_advanced/services/intune_service.py

```
@@ -0,0 +1,362 @@
+        try:
+            with zipfile.ZipFile(intunewin_path, 'r') as z:
+                # Find detection.xml
+                # Structure is usually IntuneWinPackage/Metadata/Detection.xml
+                # But let's find it.
+                names = z.namelist()
+                det_xml = next((n for n in names if n.endswith("Detection.xml")), None)
+                if not det_xml:
+                    raise ValueError("Detection.xml not found in .intunewin package")
+
+                with z.open(det_xml) as f:
+                    tree = ET.parse(f)
```

Ruff (0.14.8)

Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)